bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/ItemTriggers.json

```diff
@@ -128,6 +138,40 @@
                "required":true
             },
             {  
+               "name":"oldState",
```

Comment:
Right, so please remove it here, because this is the definition of the ItemStateUpdateTrigger.